src/lomake2.json

```diff
@@ -0,0 +1,299 @@
+[
```

Comment:
Is this file needed? If so, it should be named better.

Reply:
I don't know. It's from your commit :)

Reply:
Then you can delete the file.